### DISCUSSION TITLE
v1.1.3

### DISCUSSION THREAD

**Discussion by mxcl** [pkgxdev/pkgx#906] [Announcements]
**Full Changelog**: https://github.com/pkgxdev/pkgx/compare/v1.1.2...v1.1.3

<hr /><em>This discussion was created from the release <a href='https://github.com/pkgxdev/pkgx/releases/tag/v1.1.3'>v1.1.3</a>.</em>

**Comment by Edotamara26** [comment]
How Up level in github ?
Thank's

**Reply by minhdataaa** [reply]
Do you mean that Contributor role in discord ?
Contributor role Earned by anyone who has contributed to tea (through a pull request - must have your Discord handle in your GitHub profile).

**Comment by Edotamara26** [comment]
> **Full Changelog**: [v1.1.2...v1.1.3](https://github.com/pkgxdev/pkgx/compare/v1.1.2...v1.1.3)
> 
> _This discussion was created from the release [v1.1.3](https://github.com/pkgxdev/pkgx/releases/tag/v1.1.3)._

Sir?

**Comment by minhdataaa** [comment]
Great work, mxcl !

**Comment by Edotamara26** [comment]
Thank's

Pada tanggal Min, 28 Jan 2024 18.27, minhdataaa ***@***.***>
menulis:

**Comment by Robiansah** [comment]
Is there any node available?
